docs/development-guide/Pipelines.md

```diff
@@ -67,7 +67,7 @@ it does not perform any additional work.
 
 For cases where keeping updated information on existing and available replicas is necessary,
 such as during the failover procedure, it is important to have a host list provider that can re-fetch information once in a while,
-like the [Aurora host list provider](/wrapper/src/main/java/software/amazon/jdbc/plugin/AuroraHostListConnectionPlugin.java).
+like the Aurora host list provider.
```

Comment:
NIT: should we mention that this aurora host list provider is available using the right dialect?

Reply:
Hmm I updated it to say that it is available when connecting to aurora dbs.